systemvm/debian/etc/systemd/system/cloud-password-server@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Cloud password server on %I
+After=network.target local-fs.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=simple
+WorkingDirectory=/opt/cloud/bin
+ExecStart=/opt/cloud/bin/passwd_server_ip.py %I
```

Comment:
I would suggest to add 
~~~diff
+Restart=on-failure
~~~
https://www.freedesktop.org/software/systemd/man/systemd.service.html#Restart=

Probably check if this service could run as user "cloud".

Reply:
@resmo sure, I can add that. As per changing the owner of the process, please open a ticket. I would like to limit the scope, however, I admit I've already done too many new things in this PR :)

Reply:
yep, putting it on the list "nice to have" for later

Reply:
Thanks @resmo fixed now.